docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,204 @@
+
+## Example queries
+
+### Query: Fetching route details
+
+This example retrieves details about an object (e.g., Product, Brand, or Category) based on a given URL path.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Query route details" showLineNumbers copy
+
+     query LookUpUrl(
+       $urlPath: String!
+       # Use GraphQL Query Variables to provide a path
+     ) {
+       site {
+         route(path: $urlPath) {
+           node {
+             __typename
+             id
+             # A different response is returned based on which type of object was matched
+             ... on Category {
+               name
+               description
+             }
+             ... on Brand {
+               name
+               defaultImage {
+                 url(width: 200)
+               }
+             }
+             ... on Product {
+               name
+               description
+               images {
+                 edges {
+                   node {
+                     url(width: 500, height: 500)
+                   }
+                 }
+               }
+             }
+             ... on Redirect {
+               # all redirect attributes
+               toUrl
+             }
+           }
+         }
+       }
+     }
+```
+```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "urlPath": "/blog/"
+    }
+```
+  </Tab>
+  <Tab>
+    ```json filename="Example response" showLineNumbers copy
+     {
+        "data": {
+          "site": {
+            "route": {
+              "node": {
+                "__typename": "Blog",
+                "id": "QmxvZzoz"
+              }
+            }
+          }
+        }
+      }
+   ```
+</Tab>
+</Tabs>
+
+### Query: Using `IGNORE` redirect behavior
+
+In this example query, the `node` value is "null" because the redirect is ignored.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+   ```graphql filename="Query using IGNORE redirect behavior" showLineNumbers copy
+
+    query {
+      site {
+         route(path: "/mug" redirectBehavior: IGNORE) {
+           redirect {__typename to {__typename ...on ProductRedirect{entityId path}} toUrl} 
+           node {
+             id
+             ... on Product {
+               name
+               }
+              }
+             }
+           }
+         }
+      ```
+</Tab>
+  <Tab>
+  ```json filename="Example response" showLineNumbers copy
+     {
+  "data": {
+    "site": {
+      "route": {
+        "redirect": {
+          "__typename": "Redirect",
+          "to": {
+            "__typename": "ProductRedirect",
+            "entityId": 132,
+            "path": "/austin-coffee-cup/"
+          },
+          "toUrl": "https://store.mybigcommerce.com/austin-coffee-cup/"
+        },
+        "node": null
+      }
+    }
+  }
+}
+```
+</Tab>
+</Tabs>
+
+### Query: Using `FOLLOW` redirect behavior
+
+In this example, the `node` returns the target entity, in this case a product.
```

Comment:
> In this example, the `node` returns the entity that is targeted by the dynamic redirect as part of the response because we chose to `FOLLOW` the redirect.